packages/react/src/progress-indicator/ProgressIndicatorItem.tsx

```diff
@@ -159,69 +64,116 @@ const ProgressIndicatorItem = ({
 				},
 			}}
 		>
-			<Flex
-				as={as}
-				aria-current={isActive || undefined}
-				gap={1}
+			<Box
+				as="span"
+				columnGap={0.5}
 				css={{
-					width: '100%',
+					display: 'grid',
+					gridTemplateColumns: 'min-content 1fr',
 					textDecoration: 'none',
+					width: '100%',
 					[`[${progressIndicatorItemRingDataAttr}]:before`]: {
 						backgroundColor: backgroundColorMap[background],
 					},
-					[listItemLinkTextSelector]: {
-						fontWeight: isActive ? 'bold' : 'normal',
-					},
-					'&:hover': {
-						backgroundColor: hoverColorMap[background],
-						[`[${progressIndicatorItemRingDataAttr}]:before`]: {
-							backgroundColor: hoverColorMap[background],
-						},
-						[listItemLinkTextSelector]: packs.underline,
-					},
 				}}
 				focus
-				{...props}
 			>
 				<ProgressIndicatorItemIcon status={status} isActive={isActive} />
+
 				<Stack
 					{...{ [progressIndicatorItemTextContainerDataAttr]: '' }}
-					as="span"
-					flexDirection="column-reverse"
+					aria-current={isActive || undefined}
+					as={as}
+					css={{
+						textDecoration: 'none',
+						'&:hover': {
+							backgroundColor: hoverColorMap[background],
+						},
+						'&:hover span:not(:last-of-type)': {
+							...packs.underline,
+						},
+					}}
 					flexGrow={1}
+					fontFamily="body"
 					gap={0.25}
 					justifyContent="center"
+					paddingX={0.5}
 					paddingY={1}
-					fontFamily="body"
-					fontWeight={isActive ? 'bold' : 'normal'}
-					borderBottom
-					borderColor="muted"
+					{...(!items && { borderBottom: true, borderColor: 'muted' })}
+					{...props}
 				>
-					<Text {...{ [progressIndicatorItemTextDataAttr]: '' }}>
-						{children}
-					</Text>
 					<Text
-						color="muted"
-						fontSize="xs"
-						lineHeight="nospace"
-						css={{ textDecoration: 'none' }}
+						fontWeight={isActive ? 'bold' : 'normal'}
+						{...{ [progressIndicatorItemTextDataAttr]: '' }}
 					>
-						{label}
+						{children}
+					</Text>
+
+					<Text color="muted" fontSize="xs" lineHeight="nospace">
+						{statusLabel}
 					</Text>
 				</Stack>
-			</Flex>
+
+				{isActive && items?.some((item) => item.isActive) && (
+					<>
+						<ProgressIndicatorItemTimeline />
+
+						<Box as="ul">
+							<Box as="li" css={{ position: 'relative' }}>
+								<Flex
+									{...items.find((item) => item.isActive)}
+									alignItems="center"
+									aria-current={true}
+									as={as}
+									borderBottom
+									borderColor="muted"
+									color="text"
+									css={{
+										textDecoration: 'none',
+										'&:hover': {
+											backgroundColor: hoverColorMap[background],
+											...packs.underline,
+
+											// Extra highlighted space on hover for visual balance (not when focused which avoids hover styles covering the focus ring)
+											':not(:focus-visible)::before': {
+												backgroundColor: hoverColorMap[background],
+												content: '""',
+												height: '0.5rem',
```

Comment:
Things to note, but if we do a "tighter" spacing scale variant, this won't be right.  I think I like rendering a Box with appropriate props instead of a pseudo

Reply:
Are you saying change this to use the spacing token?

Reply:
Either a spacing token, or an actual element that uses the tokens

Reply:
- [ ] spacing token